src/components/TimeseriesChart.js

```diff
@@ -58,6 +58,12 @@ class TimeseriesChartComponent extends Component {
     this.updateTimeseries();
   }
 
+  observationType = (timeseriesUuid) => {
+    const timeseries = this.props.timeseriesEvents.find(event => event.uuid === timeseriesUuid);
+
+    return timeseries && timeseries.observation_type;
+  }
+
```

Comment:
Fails if the timeseries doesn't have any events in this time period.

It should really be looked in the timeseries object from state.timeseries[timeseriesUuid], but we don't pass that as a prop anymore because it seemed to be unnecessary (and passing it could cause unnecessary rerenders if that whole thing changed for a reason unrelated to this tile).

The only reason we care about the observationtype of an alarm is to know whether its value relates to axis "y" or to axis "y2".

The component gets *all* alarms as properties, and then in alarmReferenceLines it tries to figure out which ones are relevant for this tile.

That code could be moved to mapStateToProps instead, and then it could also divide the alarms into two groups, one of them related to axis y and the other to axis y2 (because which observationtype belongs to which axis is also decided in mapStateToProps). Makes the code simpler instead of adding this strange function back in.

Reply:
@Derryrover @remcogerlich 
Both timeseries alarm and raster alarm have an observation type. We can use this observation type to determine which Y axis to plot data. This way we don't need to get the timeseries object from Redux store back to the TimeseriesChart component anymore.